gdraw/gresource.c

```diff
@@ -733,7 +729,7 @@ static void _GResourceFindFont(const char *resourcename, GResFont *font, int is_
     char *rstr;
     memset(&rq, 0, sizeof(rq));
     if ( is_value )
-	rstr = rstr!=NULL ? copy(resourcename) : NULL;
+	rstr = copy(resourcename);
```

Comment:
@skef  can you check if this makes sense?  The prior conditional on rstr was definitely broken but not sure if the intent is to always just take the copy of resourcename

Reply:
Yes, I think this makes sense. Each GResFont should have its own `rstr` value for default initialization. The idea is to use resourcename if there is one (as that comes from a user's choice) and otherwise use the default. That's what the code starting at 755 handles -- fallback to the default when either there is no resourcename value or trying to request a font with it didn't work.